pkg/sql/opt/testutils/opttester/opt_tester_test.go

```diff
@@ -30,6 +30,10 @@ func TestOptTester(t *testing.T) {
 			// Skip any .json files; used for inject-stats
 			return
 		}
+		if strings.HasSuffix(path, ".sql") {
+			// Skip any .sql files; used for statement-bundle
```

Comment:
nit: Would it be better to have a `bundles` directory that stores each bundle, and we can ignore `bundles`? Ignoring that might be harder than a `.sql` suffix, huh?